pytorch_toolkit/object_reidentification/vehicle_reidentification/README.md

```diff
@@ -0,0 +1,131 @@
+# Vehicle Re-Identification
+
+This repository contains training and inference code for vehicle re-identification neural networks. The networks are based on the [OSNet](https://arxiv.org/abs/1905.00953) architecture provided by the [deep-object-reid](https://github.com/opencv/deep-object-reid.git) project. The code supports conversion to the ONNX\* format.
+
+| Model Name | VeRi-776\* rank-1 | VeRi-776\* mAP | GFlops | MParams | Links |
+| --- | --- | --- | --- | --- | --- |
+| vehicle-reid-0001 | 96.31 | 85.15 | 2.643 | 2.183 | [shapshot](https://download.01.org/opencv/openvino_training_extensions/models/person_reidentification/person-reidentification-retail-0270.pt), [configuration file](configs/vehicle-reid-0001.yaml) |
```

Comment:
I guess here should be a link to the snapshot for vehicle-reid-0001 model, but not to person-reidentification-retail-0270.pt

Reply:
@AlexanderDokuchaev hello, would you mind comment @sovrasov message about link to trained model? I can't find it anywhere. 
Thanks in advance!